src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionResolveHandler.cs

```diff
@@ -158,6 +178,18 @@ public async Task<CompletionItem> HandleRequestAsync(CompletionItem request, Cli
             }
 
             return resolvedCompletionItem;
+
+            static string ConvertEditIndentationToTabs(string textEdit, int tabSize)
+            {
+                var indentationString = new StringBuilder();
+                for (var i = 0; i < tabSize; i++)
+                {
+                    indentationString.Append(' ');
+                }
```

Comment:
`var indentationString = new string(' ', tabSize);` for more succinctness. My grandfather always said "Never use a big word where a diminutive one will suffice."

Reply:
Wise words 😄 thanks for the simplification!